public/stylesheets/site/2.0/05-region-main.css

```diff
@@ -21,7 +21,8 @@ http://otwcode.github.com/docs/front_end_coding/patterns/supertypes
 }
 
 #main.dashboard {
-  margin: 0.5em auto 1em 15em;
+  padding-left: 2em;
```

Comment:
The padding on `#main` is normally 2.5em on the left and right, but because of the margins here on the padding on `#dashboard`, there's actually *too* much space between the sidebar and the rest of the content. Knocking about 7px off makes the layout more balanced and buys room for the filters.